project/evaluations.html

```diff
@@ -7,7 +7,11 @@
     <link rel="stylesheet/less" type="text/css" href="assets/common.less">
     <script src="assets/less.min.js" type="text/javascript"></script>
     <script src="src/views/common/common.js" type="text/javascript"></script>
-		<script src="src/views/evaluations/evaluations.js" type="text/javascript"></script>
+    <script src="src/views/evaluations/evaluations.js" type="text/javascript"></script>
```

Comment:
You have to get to the point in which there is only one index.html file so logic like this needs to be moved accordingly.